packages/opentelemetry-tracing/src/types.ts

```diff
@@ -44,6 +50,19 @@ export interface TracerConfig {
   traceParams?: TraceParams;
 }
 
+/**
+ * Configuration options for registering the API with the SDK.
+ * Undefined values may be substituted for defaults, and null
+ * values will not be registered.
+ */
+export interface SDKRegistrationConfig {
+  /** Propagator to register as the global propagator */
+  propagator?: HttpTextFormat | null;
+
+  /** Context manager to register as the global context manager */
+  contextManager?: ScopeManager | null;
```

Comment:
should this be `undefined` instead of `null`

Reply:
same answer